examples/e2e-tests/kubernetes/release/default/definition.json

```diff
@@ -50,7 +50,8 @@
       "OSDiskSizeGB": 200,
       "vnetSubnetId": "/subscriptions/SUB_ID/resourceGroups/RG_NAME/providers/Microsoft.Network/virtualNetworks/VNET_NAME/subnets/SUBNET_NAME",
       "firstConsecutiveStaticIP": "10.239.255.239",
-      "vnetCidr": "10.239.0.0/16"
+      "vnetCidr": "10.239.0.0/16",
+      "availabilityZones": ["1", "2"]
```

Comment:
This change means we are validating this change in this PR (i.e., we are setting the cluster config that PRs run E2E against for Availability Zones 1 and 2 on master VMs only, but not on any node pools. Previously this was not allowed.